bin/ares-device.js

```diff
@@ -78,6 +81,8 @@ if (argv.level) {
 
 const options = {
     device: argv.device,
+    display : argv.display || 0,
+    inputPath : argv.argv.remain[0] || null,
```

Comment:
In help message, this is named output path.
I think the two are the same. So I want the two to have the same name.

Reply:
I agree with you. Fixed. Done.